test/Constraints/patterns.swift

```diff
@@ -291,9 +291,7 @@ switch staticMembers {
   case .init(0): break
   case .init(_): break // expected-error{{'_' can only appear in a pattern}}
   case .init(let x): break // expected-error{{cannot appear in an expression}}
-case .init(opt: 0): break // expected-error{{value of optional type 'StaticMembers?' must be unwrapped to a value of type 'StaticMembers'}}
-  // expected-note@-1{{force-unwrap using '!' to abort execution if the optional value contains 'nil'}}
-  // expected-note@-2{{coalesce using '??' to provide a default when the optional value contains 'nil'}}
+  case .init(opt: 0): break // expected-error{{pattern cannot match values of type 'StaticMembers'}}
```

Comment:
Looks like removing `UR_LabelingMismatch` and related changes fixed this diagnostic back to diagnose as `missing force unwrap`.